mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.ui/src/org/eclipse/mylyn/gitlab/ui/GitlabTaskEditorCommentPart.java

```diff
@@ -0,0 +1,402 @@
+package org.eclipse.mylyn.gitlab.ui;
```

Comment:
Shouldn't we move it to "internal" package?

Reply:
done with next commit